### PR TITLE
Fix page links in gsoc2022

### PR DESCRIPTION
**Pull request by CITIZENDOT**

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2022ideas.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ tags: gsoc
 permalink: /initiatives/gsoc/gsoc2022ideas
 ---
 
-# {{page.title}}
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2022ideas.md:8]: This was correct it’s using a front matter variable

File: pages/initiatives/gsoc/gsoc2022.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ their choice.
 Are you a student and want to code for an OWASP project? Here are the
 steps and some tips on getting started:
 
-1. Think of a good idea – For reference from last year see [GSoc {{page.year}} Ideas](gsoc{{page.year}}ideas).
+1. Think of a good idea – For reference from last year see [GSoc 2021 Ideas](gsoc2021ideas).
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2022.md:54]: This should use page.year -1 (dunno exact syntax off the top of my head)

**Comment by kingthorin** [comment]
There’s info here in setting up a local test env: https://owasp.org/migration/

File: pages/initiatives/gsoc/gsoc2022.md (hunk 2 above)
Comment (CITIZENDOT) [reply pages/initiatives/gsoc/gsoc2022.md:54]: Oh, I came to this page from google search results and found this odd. I checked the [gsoc2021ideas.md](https://github.com/OWASP/www-community/blob/master/pages/initiatives/gsoc/gsoc2021ideas.md) and submitted the PR according to that. Shall I close this now?

File: pages/initiatives/gsoc/gsoc2022ideas.md (hunk 1 above)
Comment (CITIZENDOT) [reply pages/initiatives/gsoc/gsoc2022ideas.md:8]: Oh, I came to this page from google search results and found this odd. I checked the [gsoc2021ideas.md](https://github.com/OWASP/www-community/blob/master/pages/initiatives/gsoc/gsoc2021ideas.md) and submitted the PR according to that. Shall I close this now?

Comment (kingthorin) [reply pages/initiatives/gsoc/gsoc2022ideas.md:8]: No don't close it. Let's just fix it properly ;)

**Review by kingthorin — APPROVED** [review]

**Pull request merged**